docs/blueprints/EP000.rst

```diff
@@ -0,0 +1,153 @@
+
+##########
+Motivation
+##########
+Kytos Project uses the word 'blueprint' to refer to a very specific type of document that plays a very important role in the development process. Anyone who is not familiar with the concept should be able to understand and start a draft blueprint using the EP0 (blueprint 00).
+
+
+###################
+What is a blueprint
+###################
+A blueprint is a document used to specify the details of any process, features and anything else that is desirable to be implemented in Kytos Project and cannot be reduced to a Github issue or a meeting to be outlined.
+
+#########
+Audience
+#########
+The primary audience for EPs are the core developers, as well as network engineers involved in the Kytos Project. However, other parts of the Kytos community may use the EPs to understand many aspects of the project.
+
+#########
+Rationale
+#########
+This meta blueprint is intended to make the contributor's workflow more clear about how to write a new blueprint.
+
+#############
+Specification
+#############
+
+Workflow
+**************
+Every blueprint should follow the steps below in order to be finished:
+ 
+Outline the subject
+===================
+The process for writing a blueprint should begin by describing a new idea, a specification to something that has been defined to be implemented. It is recommended that a single blueprint contain a single proposal. The more focused the blueprint, the more successful it tends to be. The Kytos team have the right to reject or approve any proposal. If in doubt, create an issue in the Kytos Project on Github and add a comment asking if the issue could be a blueprint.
+
+Submitting a blueprint
+======================
+After creating a first blueprint by adding every necessary section (check the sections under "Sections that a blueprint should have") you are encouraged to create a pull request in Kytos repository attending the following criteria:
+
+    - The file containing the proposal must follow the naming convention "EP[number].rst", where [number] is a sequential number, e.g. "EP001.rst";
+
+    - The file must contain a header following the template available in Blueprint Header section;
+
+    - The file must be written in restructured text (RST) format like the other blueprints (as you can see at 'kytos/docs/blueprints').
+
+
+Blueprint review
+================
+
+In the following days after you submit your pull request, the Kytos team will review the document adding comments and suggestions. So the author of the pull request must pay attention to the Kytos team feedback to make the review a quick process.
+
+
+Sections that a blueprint should have
```

Comment:
You are suggesting those sections, but this own blueprint BP000 is not following this recommendation.

Reply:
Well, I think all those sections won't be always necessary.  What do you think? For instance, I don't think that 'Security Implications' makes sense in BP000.
The sections that are not included in BP000 are:
-Backwards Compatibility
-Security Implications
-Rejected Ideas
Do you think they should be included?

Reply:
IMO, all sections must be there for easy navigation, even if empty or with some strings like "not applicable here".